### ISSUE TITLE
Re: Background Colour Tabs

### ISSUE THREAD

**Issue by daffydock** [madprops/grasshopper#53]
Hi, 
Maybe I have missed it. Noticed an option for changing the text on specific tabs. Say, change the colour of the text to red on a pinned tab or green, for one playing a video playing. Was wondering if there was an option to do the same but for background colour, not the text. I see you can change all tabs to be a specific colour as part of the Settings. I also see you can pick a colour yourself for any of the examples above --after the tab exists-- but that requires input from the user after the tab has been opened. It would be best if it happened automatically or on the fly by itself. 

May it be possible to add the same option to colour the background of tabs by having a preset? Thus for backgrounds, not text. Same options as for the text panel is fine. Since it seems that White/Black text on a coloured tab is far easier to read than having a bunch lines of different coloured text. 

As a side note, noticed that you added the Notes option back, I do a lot of research and that is a killer feature to have. It was not in when I first installed Grasshopper, noticed the issue where you referred to having had removed it temporarily, but it is really a very useful option since not all tabs need to be bookmarked but it is great to be able to add that context for as long as it is needed while on the go. It really adds to the well roundness of Grasshopper.

**Comment by madprops** [comment]
So you want to add background color settings, same as the text color settings, but for backgrounds. Makes sense, i'll add it.

**Comment by madprops** [comment]
Added background colors to `Colorz` settings menu in `v2069`.

**Comment by daffydock** [comment]
You are very welcome. 
Here is a bit more:  It is a very useful, lean and ergonomic feature. I had used an entire different add-on in the past to essentially do the same, or I would have to bookmark them, bloating my bookmarks with limited-time, semi-useful links. It saves time and a step by not having to clean up bookmarks as often, too.

I spend too much time switching between too many content related tabs and being able to make a quick note/references helps in not having to dive into each if I need to quote something in specific. Nor needing to have some Note++ like app/service open on the side for copy/paste actions. 

Regarding the issue in question, I took me a second to figure it out but indeed, it does exactly what I wanted as end result.
Having both background and text colour be user configurable will make for some interestingly looking setups. 
Thank you. 

Will close this issues as completed.
